include/glow/Graph/Nodes.h

```diff
@@ -203,6 +203,9 @@ inline ShapeHWD calculate3DConvPoolOutputDims(
 /// Modes of the padding operation.
 enum PaddingMode { CONSTANT = 0, REFLECT, EDGE };
 
+/// Convolution Layouts.
+enum ConvolutionLayout { NHWC = 0, NCHW };
```

Comment:
since this is applicable to pools as well, i could see this as a TensorLayout.
Slight downside in that case is that TensorLayout might be too generic and overloaded term.

Reply:
I think TensorLayout is too general, but can't think of a better name :/